imports/plugins/included/product-detail-simple/client/containers/social.js

```diff
@@ -27,11 +27,11 @@ class ProductSocialContainer extends Component {
 function composer(props, onData) {
   const product = ReactionProduct.selectedProduct() || {};
   const selectedVariant = ReactionProduct.selectedVariant() || {};
-  let title = product.title;
+  let { title } = product;
   let mediaUrl;
 
   if (selectedVariant) {
-    title = selectedVariant.title;
+    ({ title } = selectedVariant);
```

Comment:
@impactmass 
While I understand why we're doing this, I think this pattern will need a bit of documentation.

Reply:
here's the pattern:
```
let a = "" // some default value

if (ref && ref.something) { // some of the checks here makes it not easy to refactor the whole block
  a = ref.x.a
}
```